app/helpers/home_helper.rb

```diff
@@ -42,10 +42,8 @@ def account_link_to(account, button = '', path: nil)
   def obscured_counter(count)
     if count <= 0
       0
-    elsif count == 1
-      1
     else
-      '1+'
+      count
```

Comment:
That completely negates the purpose of `obscured_counter`

Reply:
Sorry, this was meant to be
```
def obscured_counter(count)
    if count <= 0
      0
    elsif count == 1
      1
    else
      count
```

Reply:
Wait, actually, upon further review, I didn't mean for this change to be there. At all.